### ISSUE TITLE
 Accessibility and post title links

### ISSUE THREAD

**Issue by ClareBryden** [picandocodigo/List-Category-Posts#464]
(If you want to submit a feature request, delete the sections above and describe the suggested feature.)

Hi - I've suggested this before at https://wordpress.org/support/topic/accessibility-and-post-title-links/ but that was probably not the correct forum. It got no response anyway. So I'm trying again here.

I’ve been running a site through the WAVE Chrome and Firefox extensions – https://wave.webaim.org/ – to check it meets accessibility guidelines. The following refers to https://gittisham.org.uk/parish-council/meetings/.

One of the things that WAVE picked up on was the redundant title text in the links: “Title attribute text is the same as text or alternative text.” Your doc illustrates it with this eg where Post Title is repeated.

`<a href="http://localhost:8080/?p=1267" title="Post Title">Post Title</a>`

Could you add a shortcode parameter to turn this title attribute off?

Thanks, Clare

**Comment by picandocodigo** [comment]
Hi Clare,
Thanks for submitting this, sounds like a good feature to add and potentially should be off by default if it's an accessibility issue!

We'll fix this.
Thanks!

**Comment by ClareBryden** [comment]
Great thanks. Off by default sounds like a good idea. Pls let me know when it's done so I can make sure the plugin updates properly.

**Comment by klemens-st** [comment]
I think we should just delete the title attribute, it serves no purpose when it's the same as the `<a>` element's text.
